### PR TITLE
Upgrade numpy, numba, hdbscan in tests

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Coooool!!
I did not see this release.

**Comment by samuelgarcia** [comment]
For the tests somehow Cython is missing.

File: requirements_test.txt [hunk 1]
```diff
@@ -5,9 +5,8 @@ pytest-cov
 # numba and hdbscan and numpy internal API
 
 # tridesclous
-numpy==1.22
-Cython
-numba==0.55.2
+numpy==1.21
```
Comment (samuelgarcia) [inline requirements_test.txt:8]: why ?

Comment (alejoe91) [reply requirements_test.txt:8]: a bunch of packages failing to install unfortunately. Let's wait some more (unless you have time to debug it ;) )

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**